mne/viz/epochs.py

```diff
@@ -953,6 +956,27 @@ def _plot_traces(params):
         ax.set_yticklabels(labels, fontsize=12, color='black')
     else:
         ax.set_yticklabels(tick_list, fontsize=12)
+
+    if len(params['events']) > 0:  # vertical lines for events.
+        for ev_line in params['ev_lines']:
+            ax.lines.remove(ev_line)  # clear the view first
+        params['ev_lines'] = list()
+        t_zero = np.where(epochs.times == 0.)[0]  # idx of 0s
+        if len(t_zero) == 0:
+            t_zero = epochs.times[0] * -1000  # extra samples if tmin > 0
```

Comment:
if `t_zero` is in samples then this should be turned to int. And `-1000` should be sampling rate.